jenkins/e2e-image/e2e-runner.sh

```diff
@@ -74,10 +74,17 @@ fi
 
 
 # We get the Kubernetes tarballs unless we are going to use old ones
+if [[ -n "${RAW_EXTRACT:-}" ]]; then
+  echo 'RAW_EXTRACT is set, --extract set by $@'
+  # TODO(fejta): delete everything after here
 if [[ "${JENKINS_USE_EXISTING_BINARIES:-}" =~ ^[yY]$ ]]; then
-  echo "Using existing binaries; not cleaning, fetching, or unpacking new ones."
+  echo 'ERROR: JENKINS_USE_EXISTING_BINARIES no longer supported'
+  echo 'Send --extract=none to scenarios/kubernetes_e2e.py'
```

Comment:
does anything use `--extract=none`? it looks like kubetest doesn't even handle it right now?